packages/core/admin/admin/src/utils/createRoute.tsx

```diff
@@ -5,8 +5,13 @@ import * as React from 'react';
 import { LoadingIndicatorPage } from '@strapi/helper-plugin';
 import { Route } from 'react-router-dom';
 
+export type TModule = () =>
```

Comment:
I'm really nitpicking and it shouldn't block the PR but I'm not sure we need to prepend this with T because it's not a generic?